src/output/index.js

```diff
@@ -91,7 +92,8 @@ export class StreamOutput {
   }
 
   log (data: string, ...args: any[]) {
-    let msg = data ? util.format(data, ...args) : ''
+    let timestamp = process.env.HEROKU_TIMESTAMPS ? `[${moment().format()}] ` : ''
```

Comment:
also, we should make sure it works for `write` as well. I think what you could do is put this into `write` and add a static variable `startOfLine` (so it's the same for both stdout and stderr) that is used like this:

```js
static startOfLine = true
write (msg: string) {
  if (this.constructor.startOfLine) msg = `[${moment().format()}]${msg}`
  this.constructor.startOfLine = msg.endsWith('\n')
}
```

I think that should work